examples/platform/silabs/MatterConfig.cpp

```diff
@@ -312,6 +316,21 @@ CHIP_ERROR SilabsMatterConfig::InitMatter(const char * appName)
     // Init Matter Server and Start Event Loop
     err = chip::Server::GetInstance().Init(initParams);
 
+    // [sl-only]: Configure Wi-Fi App Sleep Manager
+#if SL_MATTER_ENABLE_APP_SLEEP_MANAGER
+    err = app::Silabs::ApplicationSleepManager::GetInstance()
+              .SetFabricTable(&Server::GetInstance().GetFabricTable())
```

Comment:
Rather than all those individual sets, can't the Init function take those as parameters? Is there any other instance in which we would set these outside of Init?